.github/workflows/mint/nginx-4-node.conf

```diff
@@ -23,10 +23,9 @@ http {
     # include /etc/nginx/conf.d/*.conf;
 
     upstream minio {
-        server minio1:9000;
-        server minio2:9000;
-        server minio3:9000;
-        server minio4:9000;
+        server minio1:9000 max_fails=1 fail_timeout=10s;
+        server minio2:9000 max_fails=1 fail_timeout=10s;
+        server minio3:9000 max_fails=1 fail_timeout=10s;
```

Comment:
Removed minio4 since we are taking it down and pausing it. We add this to ensure that downed node is skipped however we test if other nodes respond properly.